### PR TITLE
Add rTorrent Flood plugin

### PR DESCRIPTION
**Pull request by pierrickrouxel**
Adds the rTorrent with Flood UI as plugin

### CODE REVIEW

**Comment by miwi-fbsd** [comment]
Hi,

I have open a pull request to make the shell script executable
https://github.com/pierrickrouxel/iocage-plugin-rtorrent-flood/pull/1/commits.

Once thats pulled in we can add the plugin

**Comment by pierrickrouxel** [comment]
It’s merged. Thank you.

File: INDEX [hunk 1]
```diff
@@ -325,5 +325,13 @@
         "description": "DVR server for the Channels apps. Makes digital recordings of broadcast television using HDHomeRun network tuners. (Requires Subscription)",
         "primary_pkg": "",
         "official": false
+    },
+     "rtorrent-flood": {
+        "MANIFEST": "rtorrent-flood.json",
+        "name": "rtorrent-flood",
+        "icon": "https://www.trueos.org/iocage-icons/rtorrent-flood.json",
```
Comment (miwi-fbsd) [inline INDEX]: I just realise that this needs to be an icon hosted somewhere at your side.

File: rtorrent-flood.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    "tmux",
+    "rtorrent",
+    "openvpn",
+    "git",
+    "node",
+    "npm"
+  ],
+  "properties": {
+    "allow_raw_sockets": "1",
+    "vnet": "on",
```
Comment (miwi-fbsd) [inline rtorrent-flood.json]: This can be removed its now automatically set

File: rtorrent-flood.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  "pkgs": [
+    "tmux",
+    "rtorrent",
+    "openvpn",
+    "git",
+    "node",
+    "npm"
+  ],
+  "properties": {
+    "allow_raw_sockets": "1",
```
Comment (miwi-fbsd) [inline rtorrent-flood.json]: 1 without ""

File: rtorrent-flood.json [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    "vnet": "on",
+    "allow_tun": "1"
```
Comment (miwi-fbsd) [inline rtorrent-flood.json]: 1 without ""

File: INDEX [hunk 5]
```diff
@@ -309,7 +309,7 @@
         "description": "Network infrastructure management for small networks: DNS, DHCP, router advertisement, and network boot.",
         "official": false,
         "primary_pkg": "dnsmasq"
-    },
+     },
```
Comment (sonicaj) [inline INDEX]: Please make to sure that we don't have formatting issues with json, this space is unnecessary here

File: INDEX [hunk 6]
```diff
@@ -325,5 +325,13 @@
         "description": "DVR server for the Channels apps. Makes digital recordings of broadcast television using HDHomeRun network tuners. (Requires Subscription)",
         "primary_pkg": "",
         "official": false
+    },
```
Comment (sonicaj) [inline INDEX]: This also seems out of place, it needs to be indented with 4 spaces. And the same is true for the lines below

File: rtorrent-flood.json [hunk 7]
```diff
@@ -0,0 +1,28 @@
+{
+  "name": "rtorrent-flood",
```
Comment (sonicaj) [inline rtorrent-flood.json]: Same is true for this json file as well please. We need to indent with 4 spaces, you could perhaps use some tool/language to dump this json indenting with 4 spaces at the end

File: INDEX (hunk 1 above)
Comment (pierrickrouxel) [reply INDEX]: I did a mistake. It's https://www.trueos.org/iocage-icons/rtorrent-flood.png.
I pushed it to the repository. You will not expose it ?

Comment (miwi-fbsd) [reply INDEX]: yes we can do that.

**Comment by pierrickrouxel** [comment]
I think it's ok.

**Comment by miwi-fbsd** [comment]
@pierrickrouxel thank you :)

**Pull request merged**